src/test/kotlin/uk/gov/justice/digital/hmpps/assessments_events/integration/service/EventsServiceTest.kt

```diff
@@ -0,0 +1,98 @@
+package uk.gov.justice.digital.hmpps.assessments_events.integration.service
+
+import com.amazonaws.services.sqs.model.Message
+import com.amazonaws.services.sqs.model.PurgeQueueRequest
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.assertj.core.api.Assertions.assertThat
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.matches
+import org.awaitility.kotlin.untilCallTo
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.test.context.jdbc.Sql
+import org.springframework.test.context.jdbc.SqlConfig
+import org.springframework.test.context.jdbc.SqlGroup
+import uk.gov.justice.digital.hmpps.assessments_events.dto.EventDto
+import uk.gov.justice.digital.hmpps.assessments_events.dto.EventType
+import uk.gov.justice.digital.hmpps.assessments_events.integration.IntegrationTestBase
+import java.time.Duration
+import java.time.LocalDateTime
+
+@SqlGroup(
+  Sql(
+    scripts = ["classpath:assessment/before-test.sql"],
+    config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED)
+  ),
+  Sql(
+    scripts = ["classpath:assessment/after-test.sql"],
+    config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED),
+    executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
+  )
+)
+class EventsServiceTest : IntegrationTestBase() {
```

Comment:
Full E2E test against LocalStack where we are getting the 3 events after date '20/06/2016' that we have in the in memory database filled with the sql scripts data. There are 1 Completed assessment and 2 guillotined.

`INSERT INTO OASYS_SET (OASYS_SET_PK, OASYS_ASSESSMENT_GROUP_PK, ASSESSMENT_TYPE_ELM, ASSESSMENT_STATUS_ELM,  DATE_COMPLETED)
VALUES (5432, 6543, 'LAYER_3', 'COMPLETE', TO_DATE('2018-06-20 23:00:09', 'YYYY-MM-DD HH24:MI:SS')),
       (5433, 6543, 'LAYER_3', 'OPEN', null),
       (5434, 6543, 'LAYER_3', 'GUILLOTINED', TO_DATE('2016-07-20 02:00:09', 'YYYY-MM-DD HH24:MI:SS')),
       (5435, 6543, 'LAYER_3', 'GUILLOTINED', TO_DATE('2016-07-20 10:00:09', 'YYYY-MM-DD HH24:MI:SS'))`